internal/provider/resource_auth0_connection.go

```diff
@@ -23,6 +23,7 @@ var connectionSchema = map[string]*schema.Schema{
 	"display_name": {
 		Type:        schema.TypeString,
 		Optional:    true,
+		Computed:    true,
```

Comment:
> The following option fields on the connection authorization_endpoint, issuer, jwks_uri, token_endpoint, or user_info_endpoint can be set to default values by the API

`display_name` as well?

Reply:
Apparently `display_name` is a computed value for OIDC connection types, defaulting to the `name` field if not defined. It's a bit of a toss-up of wether its' worth it to mark as computed for all connections to accommodate this single connection type, but the consequences are minimal either way. @sergiught and I just changed our minds on this and decided to remove to minimize the effects for the majority of users.